src/dwalk/dwalk.c

```diff
@@ -62,71 +114,83 @@ static int print_flist_distribution(struct distribute_option *option, mfu_flist*
     for (int i = 0; i < size; i++) {
          /* get the size of the file */
          uint64_t file_size = mfu_flist_file_get_size(flist, i);
-         
+
          /* loop through the bins and find the one the file belongs to,
-          * set last bin to -1, if a bin is not found while looping through the 
+          * set last bin to -1, if a bin is not found while looping through the
           * list of file size separators, then it belongs in the last bin
           * so (last file size - MAX bin) */
          int max_bin_flag = -1;
          for (int j = 0; j < separators; j++) {
              if (file_size <= option->separators[j]) {
                  /* found the bin set bin index & increment its count */
                  dist[j]++;
- 
-                 /* a file for this bin was found so can't belong to 
+
+                 /* a file for this bin was found so can't belong to
                   * last bin (so set the flag) & exit the loop */
                  max_bin_flag = 1;
                  break;
-             }       
+             }
          }
- 
+
          /* if max_bin_flag is still -1 then the file belongs to the last bin */
          if (max_bin_flag < 0) {
              dist[separators]++;
          }
     }
- 
+
     /* get the total sum across all of the bins */
     uint64_t* disttotal = (uint64_t*) MFU_MALLOC((separators + 1) * sizeof(uint64_t));
     MPI_Allreduce(dist, disttotal, (uint64_t)separators + 1, MPI_UINT64_T, MPI_SUM, MPI_COMM_WORLD);
- 
+
     /* Print the file distribution */
     if (rank == 0) {
          /* number of files in a bin */
          uint64_t number;
-         printf("Range\tNumber\n");
-         for (int i = 0; i <= option->separator_number; i++) {
-             printf("[");
+         double size_tmp;
+         const char* size_units;
+         printf("%-49s %-10s\n", "Range", "Number");
```

Comment:
After dropping all of the 10's below, I think this 49 would become:
```
[ 0000.00 KB - 0000.00 KB ]
```
I count 27 chars long from bracket to bracket, inclusive.